components/events/EmptyCard.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <article class="empty-card">
+    <img
+      class="empty-card__image"
+      src="data:image/svg+xml,%3Csvg width='224' height='200' viewBox='0 0 224 200' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='224' height='200' fill='%23C1C7CD'/%3E%3Cpath d='M112 111.346V81.7308M156 138H68L112 61L156 138Z' stroke='white' stroke-width='2' stroke-linejoin='round'/%3E%3Cpath d='M112 126.154C113.62 126.154 114.933 124.828 114.933 123.192C114.933 121.557 113.62 120.231 112 120.231C110.38 120.231 109.067 121.557 109.067 123.192C109.067 124.828 110.38 126.154 112 126.154Z' fill='white'/%3E%3C/svg%3E%0A"
```

Comment:
Could you add this img as a file and follow the structure we already have on the `LearningResourceCard` component?

Reply:
@y4izus - please have a look at the updates.

I added the img (as an svg to keep the clean lines). I was thinking that `image` doesn't necessarily have to be a prop, since it looks like this will be the image for all empty cards?

Thanks!

Reply:
we are going to reuse the card component, so the img is going to change :)